org.netsurf_browser.NetSurf.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 app-id: org.netsurf_browser.NetSurf
 runtime: org.freedesktop.Platform
-runtime-version: "19.08"
+runtime-version: "20.08"
```

Comment:
21.08 is out. maybe it's a good time to switch to it right now.

Reply:
Cool, I wasn't sure if it was available yet. I'll resubmit.